postgresql/ref/grant.xml

```diff
@@ -263,38 +263,37 @@ GRANT <replaceable class="parameter">nom_role</replaceable> [, ...] TO <replacea
     <literal>TRUE</literal>, donc <literal>WITH ADMIN OPTION</literal>
     est un synonyme pour <literal>WITH ADMIN TRUE</literal>. Lors de la
     modification d'une appartenance existante, l'omission d'une option
-    résultat en la conservation de la valeur actuelle.
+    résulte en la conservation de la valeur actuelle.
    </para>
 
    <para>
-    L'option <literal>ADMIN</literal> autorise le membre
-    en octroyer l'appartenance à d'autres rôles, et la révoquer. Sans cette
-    option, les utilisateurs ordinaires ne peuvent pas le faire. Un rôle ne
-    dispose pas de l'option <literal>WITH ADMIN OPTION</literal> lui-même Les
+    L'option <literal>ADMIN</literal> autorise le membre à octroyer
+    l'appartenance à d'autres rôles, et la révoquer. Sans cette option, les
+    utilisateurs ordinaires ne peuvent pas le faire. Un rôle ne dispose pas de
+    l'option <literal>WITH ADMIN OPTION</literal> sur lui-même. Les
     superutilisateurs de la base peuvent donner ou révoquer l'appartenance à un
-    rôle pour tout personne. Cette option est à
-    <literal>FALSE</literal> par défaut.
+    rôle pour toute personne. Cette option est à <literal>FALSE</literal> par
+    défaut.
    </para>
 
    <para>
-    L'option <literal>INHERIT</literal> contrôle le statut d'héritage
-    du nouveau membre&nbsp;; voir <xref linkend="role-membership"/>
-    pour les détails sur l'héritage. Si elle est configurée à
-    <literal>TRUE</literal>, elle fait que le nouveau membre hérite
-    du rôle donné. Si elle est configurée à
-    <literal>FALSE</literal>, le nouveau membre n'hérite pas. Sans
-    indication lors de la création d'un nouveau rôle, le défaut est
-    l'ajout de l'attribut d'héritage au rôle ajouté.
+    L'option <literal>INHERIT</literal> contrôle le statut d'héritage du nouveau
```

Comment:
Original dans REL_16_STABLE:

``` xml
  <para>
   The <literal>INHERIT</literal> option, if it is set to
   <literal>TRUE</literal>, causes the member to inherit the privileges of
   the granted role. That is, it can automatically use whatever database
   privileges have been granted to that role. If set to
   <literal>FALSE</literal>, the member does not inherit the privileges
   of the granted role. If this clause is not specified, it defaults to
   true if the member role is set to <literal>INHERIT</literal> and to false
   if the member role is set to <literal>NOINHERIT</literal>.
   See <link linkend="sql-createrole"><command>CREATE ROLE</command></link>.
  </para>
```